SharePointFramework/PortfolioWebParts/src/data/index.ts

```diff
@@ -428,7 +431,7 @@ export class DataAdapter {
    * * Site users
    * * Combines the data
    */
-  public async fetchEncrichedProjects(): Promise<ProjectListModel[]> {
+  public async fetchEncrichedProjects(getAll: boolean): Promise<ProjectListModel[]> {
```

Comment:
You added a parameter so update the function comment.

Reply:
Done ✅